Sources/Pulse/PersistentLogHandler.swift

```diff
@@ -0,0 +1,66 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2020 Alexander Grebenyuk (github.com/kean).
+
+import CoreData
+import Foundation
+import Logging
+
+public struct PersistentLogHandler {
+    public var metadata = Logger.Metadata()
+    public var logLevel = Logger.Level.info
+
+    /// An id of the current log sesion.
+    public private(set) var logSessionId = UUID()
```

Comment:
This should move somewhere else. As a temporary solution, it could be simply made a `static` property on `PersistentLogHandler`.

The reason this ID exists is to allow the console to distinguish between user sessions. Currently the iOS console only displays messages for the current session by default. I didn't invest too much time into designing how it should work, this might change in the future.

Reply:
I followed your suggestion and moved it into a singleton on `PersistentLogHandler`.